setup.go

```diff
@@ -65,11 +65,11 @@ var loggerMutex sync.RWMutex // guards access to global logger state
 var loggers = make(map[string]*zap.SugaredLogger)
 var levels = make(map[string]zap.AtomicLevel)
 
-// primaryFormat is the default format of the primary core used for logging
-var primaryFormat LogFormat = ColorizedOutput
+// defaultFormat is the default format of the primary core used for logging
+var defaultFormat LogFormat = ColorizedOutput
```

Comment:
I originally named them primaryFormat/primaryLevel because they correspond to the format and level used by the primary core which. It's not really a default

Reply:
primaryFormat makes sense. primaryLevel isn't quite correct because the level for the primary logger needs to be debug.